app/api/fix-unstake/route.ts

```diff
@@ -0,0 +1,39 @@
+import { stack } from "@/lib/stack/client";
+import { forEachRecord, retrieveRecords } from "./lib";
+
+export async function GET() {
+  try {
+    const data = await retrieveRecords();
+
+    let changesCount = 0;
+    await forEachRecord(data, async (stake, event, userAddress) => {
+      const metadata = { fixUnstake: true, id: stake.id };
+      const { channelAddress, fid, channelId, amount } = stake;
+
+      const results = await Promise.all([
+        // unstaking in deprecated event
+        stack.track(`channel_stake_from_${channelAddress}`, { account: channelAddress, points: -amount, metadata }),
+        stack.track(`channel_stake_to_${fid}`, { account: userAddress, points: amount, metadata }),
```

Comment:
this feels like an `unstake` event.
- stake: from `user` => `channel`
- unstake: from `channel` => `user`

are we labeling both actions as `stake`?